servicetalk-opentracing-http/src/main/java/io/servicetalk/opentracing/http/TracingHttpRequesterFilter.java

```diff
@@ -34,13 +35,21 @@
 import io.opentracing.Tracer;
 import io.opentracing.Tracer.SpanBuilder;
 
+import java.util.function.UnaryOperator;
+
 import static io.opentracing.tag.Tags.HTTP_METHOD;
 import static io.opentracing.tag.Tags.HTTP_URL;
 import static io.opentracing.tag.Tags.SPAN_KIND;
 import static io.opentracing.tag.Tags.SPAN_KIND_CLIENT;
 
 /**
  * An HTTP filter that supports open tracing.
+ * <p>
+ * Append this filter before others that are expected to to see {@link Scope} for this request/response. Filters
+ * appended after this filter that use operators with the <strong>after*</strong> prefix on the
+ * {@link StreamingHttpResponse#transformRawPayloadBody(UnaryOperator) response payload body} (e.g.
```

Comment:
This disclaimer also applied for metadata in case of error, you may want to add that.